scripts/pylib/twister/twisterlib/testplan.py

```diff
@@ -549,6 +554,46 @@ def add_testsuites(self, testsuite_filter=None):
                                 suite.platform_allow,
                                 f"platform_allow in {suite.name}")
 
+                        if self.options.log_level == "QA":
```

Comment:
this `if` seems redundant. Using logger.qa already handles the logic when to print

Reply:
ditto